tests/aws/services/lambda_/test_lambda_api.py

```diff
@@ -377,6 +377,87 @@ def test_function_lifecycle(self, snapshot, create_lambda_function, lambda_su_ro
             aws_client.lambda_.delete_function(FunctionName=function_name)
         snapshot.match("delete_postdelete", e.value.response)
 
+    @markers.aws.validated
```

Comment:
nit: Would it make sense to group these tests under a separate class rather than adding them to `TestLambdaFunction`, especially given they operate on a completely different test case?